src/lib/REST/Output/ValueObjectVisitor/ContentTree/Node.php

```diff
@@ -34,6 +34,15 @@ public function visit(Visitor $visitor, Generator $generator, $data)
         $generator->startValueElement('contentId', $data->contentId);
         $generator->endValueElement('contentId');
 
+        $generator->startValueElement('versionNo', $data->versionNo);
+        $generator->endValueElement('versionNo');
```

Comment:
And at least in all the remaining lines you change:
```suggestion
        $generator->valueElement('versionNo', $data->versionNo);
```